src/components/Exchange/FeesTooltip.tsx

```diff
@@ -0,0 +1,72 @@
+import { t, Trans } from "@lingui/macro";
+import ExternalLink from "components/ExternalLink/ExternalLink";
+import StatsTooltipRow from "components/StatsTooltip/StatsTooltipRow";
+import Tooltip from "components/Tooltip/Tooltip";
+
+type Row = {
+  label: string;
+  value: string;
+};
+
+function FeesTooltip({ totalFees, fundingFee, positionFee, swapFee, executionFee, isOpening = true }) {
+  const feeRows: Row[] = [];
+  if (!isOpening && fundingFee) {
+    feeRows.push({
+      label: t`Borrow Fee`,
+      value: fundingFee,
+    });
+  }
+  if (isOpening && swapFee) {
+    feeRows.push({
+      label: t`Swap Fee`,
+      value: swapFee,
+    });
+  }
+  if (positionFee) {
```

Comment:
1. There seems to be a duplication of conditions?

```
if(!isOpening && swapFee) {...}

if(isOpening && swapFee) {...}
```
From this it is hard to see what the list of commissions will look like

Also, here are a couple of suggestions:
2. move the formatting also in this tooltip, now in PositionSeller/Editor/SwapBox
there is a duplicated formatting code like
```
${formatAmountFree(minExecutionFee, 18, 5)} ${nativeTokenSymbol}}
```
Thus, FeesTooltip component is not fully unified

3. Is it possible to remove the serial push in Rows array? I mean, if the only problem is the different order of opening / closing scenarios, can we just use "openingFee, closingFee" props instead of positionFee and just render them where needed?
Or, alternatively, if there is too much difference between the different Modals / SwapBoxes, it is easier to use this as
```
<FeesTooltip 
 rows={...}
/>
```

Reply:
I have updated the code based on your suggestions.